afskrivining_af_foraeldede_sagsomkostninger/sap_process.py

```diff
@@ -42,16 +44,11 @@ def delete_cost(session, fp: str, aftale: str, bilag: str, dry_run=True) -> None
                 # press 'Accept' button.
                 session.FindById('wnd[1]/tbar[0]/btn[0]').press()
             row_id += 1
-
-    except:
+    except pywintypes.com_error:
```

Comment:
This comment is for the entire try-catch block:

You can pass False to session.findById to tell it to not raise an error when the object is missing.
Wrapping the entire thing in a try is not a good idea since errors could happy at other lines that we don't want to catch.

```suggestion
    if session.findById('wnd[1]/usr', False) is not None:
       do stuff with popup
```